server/controllers/chartsController.js

```diff
@@ -22,12 +22,31 @@ const postProcessData = (data) => {
       processedData[valueLabel] = [newEntry]
     }
   })
+  const largestHorizontalSection = Object.values(processedData).sort(
+    (arr1, arr2) => arr2.length - arr1.length
+  )[0]
+
+  const notAvailableArray = largestHorizontalSection.map((siteSection) => {
```

Comment:
I think it's worth having consistent language and use either study or site for variable names (this is my fault).

```suggestion
  const notAvailableArray = largestHorizontalSection.map((studySection) => {
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/123/commits/bdef2da4a95afc977ece57e85588ca68dd80b215